modules/products/rubymine/src/main/kotlin/com/metalbear/mirrord/products/rubymine/RubyMineRunConfigurationExtension.kt

```diff
@@ -48,15 +48,20 @@ class RubyMineRunConfigurationExtension : RubyRunConfigurationExtension() {
             if (isMac) {
                 this.executable = cmdLine.exePath
             }
-        }.start()?.let { (mirrordEnv, patched) ->
+        }.start()?.let { executionInfo ->
+            val mirrordEnv = executionInfo.environment
+            executionInfo.envToUnset?.let { envToUnset ->
+                for (key in envToUnset.iterator()) {
+                    mirrordEnv.remove(key)
+                }
```

Comment:
This will unset the variable only if it was fetched from the remote

Reply:
Like in Rider, `cmdLine.environment` gives us access to raw env. We can do the removal